CHANGELOG.md

```diff
@@ -1,7 +1,28 @@
 # Change Log
+
 All notable changes to this project will be documented in this file.
 See the [change log guidelines](http://keepachangelog.com/) for information on how to structure the file.
 
+## [5.1.0]
+
+### Added
+
+- Simplify the setup process with automated creation of deegree configurations and capabilities for WFS/WMS services.
+- Introduced support for the MS Access JDBC driver, expanding connectivity options for your database needs.
+- Migration to Java 17, updating Gradle Docker plugin to 9.3.1 and baseImage to Java 17, ensuring compatibility and leveraging the latest Java features.
+- Updated dependencies addressing security vulnerabilities, safeguarding your system against potential threats.
+- Added support to custom functions for merge statistics, offering more flexibility and customization options.
```

Comment:
This is a **fix** as the respective commit message states.

I would propose
"Fixed error when collecting merge statistics for an alignment with custom functions."